rpc_state_reader_sn_api/src/lib.rs

```diff
@@ -323,6 +325,24 @@ impl RpcState {
         let sequencer_address: StarkFelt =
             serde_json::from_value(block_info["result"]["sequencer_address"].clone()).unwrap();
 
+        let transactions: Vec<_> = block_info["result"]["transactions"]
+            .as_array()
+            .unwrap()
+            .iter()
+            .filter_map(|result| match result["type"].as_str().unwrap() {
+                "INVOKE" => match result["version"].as_str().unwrap() {
+                    "0x0" => Some(Transaction::Invoke(InvokeTransaction::V0(
+                        serde_json::from_value(result.clone()).unwrap(),
+                    ))),
+                    "0x1" => Some(Transaction::Invoke(InvokeTransaction::V1(
+                        serde_json::from_value(result.clone()).unwrap(),
+                    ))),
+                    _ => None,
+                },
+                _ => None,
+            })
+            .collect();
+
```

Comment:
maybe refactor that mapping into its own function, also will need to replace in `get_transaction()`.